server/controllers/games.js

```diff
@@ -0,0 +1,73 @@
+const Game = require('../models').Game;
+
+
+module.exports= {
+  create(req, res) {
```

Comment:
Added documentation to games controller. Probably not the greatest, if there are specific formats/tags beyond what I have added, feel free to add them.